client/src/components/forms/AddEventForm.tsx

```diff
@@ -79,10 +77,10 @@ const AddEventForm = ({ onSuccess, initialValues, submitText, isLoading, day }:
           onChange={formik.handleChange}
           error={formik.touched.eventDate && Boolean(formik.errors.eventDate)}
           helperText={formik.touched.eventDate && formik.errors.eventDate}
-          disabled={initialValues?.eventDate !== undefined}
+          disabled
```

Comment:
you can't set disabled always, how about when you add "Add event button" which will trigger new addEventForm the date field must be editable.

Reply:
Change name of Form to "EventForm"